packages/python/patch-no-sys-resource_h

```diff
@@ -0,0 +1,29 @@
+--- Modules/_lsprof.c
```

Comment:
I've created https://github.com/moreati/cloudabi-ports/issues/4 to record this. Would you accept the PR with this as is, for now?